extensions/wikia/TemplateDraft/TemplateDraftHooks.class.php

```diff
@@ -2,6 +2,20 @@
 
 class TemplateDraftHooks {
 
+	public static function onSkinAfterBottomScripts( $skin, &$text ) {
+		global $wgTitle, $wgUser;
```

Comment:
`$skin` should be a context source, so you should be able to avoid the globals with `$this->getTitle()` and `$this->getUser()`.